scripts/reflow_skylark

```diff
@@ -17,5 +17,6 @@
 
 buildifier $(find . -type f \
     -iname "*.bzl" -or \
-    -name "BUILD*"
-)
\ No newline at end of file
+    -name "*.bazel" -or \
```

Comment:
I think this is what's killing your .bazelignore and making buildifier edit it.

Reply:
Nope.